indico/modules/events/papers/schemas.py

```diff
@@ -228,6 +234,60 @@ class Meta(PaperSchema.Meta):
         fields = ('is_in_final_state', 'contribution', 'revisions', 'state')
 
 
+class PaperFileTypeSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = PaperFileType
+        fields = ('id', 'name', 'extensions', 'allow_multiple_files', 'required', 'publishable', 'is_used',
+                  'filename_template', 'url')
+
+    is_used = fields.Function(lambda ft: PaperFile.query.with_parent(ft).has_rows())
+    url = Function(lambda ft: url_for('.api_edit_file_type',
+                                             ft.event,
+                                             file_type_id=ft.id,
+                                             _external=True))
```

Comment:
weird indentation, I think it may all fit in one line, or alternatively let ruff format this statement